README.md

```diff
@@ -0,0 +1,21 @@
+# Jekxyl
+
+## Installation
+
+With [Composer](http://getcomposer.org/), you need to require
+[`jekxyl/jekxyl`](https://packagist.org/packages/jekxyl/jekxyl):
+
+```sh
+$ composer global require jekxyl/jekxyl
+```
+
+Please, read the website to [get more informations about how to
+install](http://hoa-project.net/Source.html).
+
+## Usage
+
+```sh
+$ jekxyl new ./MyProject
+$ jekxyl compile --source ./MyProject/Source --destination ./MyProject/Dist
```

Comment:
The `Dist` directory, because empty, was not present in the Git tree. Now it should work. Can you re-test please?